.github/workflows/php-library-security.yml

```diff
@@ -0,0 +1,57 @@
+name: Security
+
+on:
+ workflow_call:
+  secrets:
+      SNYK_TOKEN:
+        required: true
+  inputs:
+    upload-sarif:
+      required: false
+      type: boolean
+      default: false
+  
+jobs:
+ snyk:
+   runs-on: ubuntu-latest
+   name: Snyk PHP Scan
+
+   steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: 8.1
```

Comment:
Minor again, but to reduce maintenance, `setup-php` now supports the `latest` keyword - which might be useful for all instances where we're hardcoding a PHP version.